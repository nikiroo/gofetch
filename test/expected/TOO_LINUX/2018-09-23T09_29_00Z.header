0Linagora a fêté ses 18 ans cet été : la vidéo	null/TOO_LINUX/2018-09-23T09_29_00Z		70
i  2018/09/23
i
i    Cet  été,  tout comme Toolinux.com, LINAGORA a soufflé sa 18ème
i    bougie.  Une  vidéo  a été publiée pour fêter cet événement. Ce
i    18e  anniversaire  a  été  l’occasion  pour  le groupe - devenu
i    international  !  -  de  publier  une  refonte  de son site web
i    autour   de  ses  valeurs  consolidées.  Note  aux  lecteurs  :
i    Linagora finance toolinux.com depuis mai (...)
i
