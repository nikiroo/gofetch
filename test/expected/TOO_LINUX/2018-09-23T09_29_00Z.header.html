<!DOCTYPE html>
<html>
<head>
  <meta http-equiv='content-type' content='text/html; charset=utf-8'>
  <meta name='viewport' content='width=device-width, initial-scale=1.0'>
  <style type='text/css'>
    body { margin: 1em 15%; }
  </style>
</head>
<body>
<div class='story-header'>
	<h1><a href='2018-09-23T09_29_00Z.html'>Linagora a fêté ses 18 ans cet été : la vidéo</a></h1>
	<div class='details'>(2018/09/23)</div>
	<br/>
	<div class='content' style='text-align: justify'>
		Cet été, tout comme Toolinux.com, LINAGORA a soufflé sa 18ème bougie. Une vidéo a été publiée pour fêter cet événement. Ce 18e anniversaire a été l’occasion pour le groupe - devenu international ! - de publier une refonte de son site web autour de ses valeurs consolidées. Note aux lecteurs : Linagora finance toolinux.com depuis mai (...)
	</div>
<hr/>
</div>
</body>
